proposals/lip-0046.md

```diff
@@ -196,7 +196,10 @@ This function returns `True` if `seedReveal` was correctly revealed, otherwise,
 It is specified as:
 
 ```python
-def isSeedRevealValid(generatorAddress: Address, seedReveal: bytes) -> bool:
+def isSeedRevealValid(generatorAddress: Address, blockAssets: BlockAssets) -> bool:
+    let `blockAssetBytes` be the bytes included in the `blockAssets` for the Random module           
```

Comment:
Minor point: Perhaps we should check with implementation how they recover this and do it the same way for clarity.

Reply:
What do you mean by recover?

Reply:
I agree with Vikas on that we do not necessarily need to provide all the technical details here. I hope it should be clear enough for the devs.